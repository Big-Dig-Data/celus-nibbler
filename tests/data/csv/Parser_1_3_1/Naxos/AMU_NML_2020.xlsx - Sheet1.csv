Metric,Jan-2020,Feb-2020,Mar-2020,Apr-2020,May-2020,Jun-2020,Jul-2020,Aug-2020,Sep-2020,Oct-2020,Nov-2020,Dec-2020
Tracks,752,531,493,411,232,177,351,356,252,714,678,215
