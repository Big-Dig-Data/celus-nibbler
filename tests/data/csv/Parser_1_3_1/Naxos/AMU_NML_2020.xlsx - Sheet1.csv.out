"2020-01-01","2020-01-31","Naxos",,"Tracks",,,"752"
"2020-02-01","2020-02-29","Naxos",,"Tracks",,,"531"
"2020-03-01","2020-03-31","Naxos",,"Tracks",,,"493"
"2020-04-01","2020-04-30","Naxos",,"Tracks",,,"411"
"2020-05-01","2020-05-31","Naxos",,"Tracks",,,"232"
"2020-06-01","2020-06-30","Naxos",,"Tracks",,,"177"
"2020-07-01","2020-07-31","Naxos",,"Tracks",,,"351"
"2020-08-01","2020-08-31","Naxos",,"Tracks",,,"356"
"2020-09-01","2020-09-30","Naxos",,"Tracks",,,"252"
"2020-10-01","2020-10-31","Naxos",,"Tracks",,,"714"
"2020-11-01","2020-11-30","Naxos",,"Tracks",,,"678"
"2020-12-01","2020-12-31","Naxos",,"Tracks",,,"215"
